_topic_maps/_topic_map_rosa.yml

```diff
@@ -87,131 +123,128 @@ Topics:
   File: rosa-sts-deleting-access-cluster
 - Name: Deleting a ROSA cluster
   File: rosa-sts-deleting-cluster
+- Name: Deploying ROSA using Identity and Access Management
```

Comment:
@arendej and @0kashi hi! My understanding is that this subsection is now for the non-STS deployment assemblies. I proposed that we update this navigation menu title to make this clearer. When I read "Identity and Access Management" in this title, I think of the STS IAM resources.

What is your view? Would "non-STS" in the title be suitable? Thanks!